SDVModTest/i18n/default.json

```diff
@@ -1,10 +1,13 @@
 {
     "Billboard" : "Billboard",
     "Calendar" : "Calendar",
+    "Day" : "day",
     "Days" : "days",
+    "DayToMature" : "day to mature",

```

Comment:
These changes need to be added to all the language files. I'm not sure all the languages even support this idea, but I suppose ones that don't could just use the same string for both cases.
I would just use google translate to do this for now.